contents/java/Java.md

```diff
@@ -14,6 +14,161 @@ Authors: John Doe, Jane Doe
 
 ## Reflection
 
-// Reflection overview
+### What is Java reflections
+
+Java reflections allow the inspection of classes, methods and fields during runtime without having any knowledge of it
+during compile time. It is also possible to new instance of existing objects, invoke methods and changing values of
```

Comment:
Can consider adding a comma after "during runtime". It also seems like a word is missing in the second sentence btw! It might be better to write "...change values..." instead of "...changing values...".